File: competition/03-primes/primes.py
def is_prime(n):
    d = 2
    while d * d <= n:
        if n % d == 0:
            return False
        d += 1
    return True


count = 0
for n in range(2, 300000):
    if is_prime(n):
        count += 1
print(count)

File: competition/03-primes/primes.rb
def is_prime(n)
  d = 2
  while d * d <= n
    return false if n % d == 0
    d += 1
  end
  true
end

count = 0
(2...300000).each do |n|
  count += 1 if is_prime(n)
end
puts count
